libr/enco/d/pokered

```diff
@@ -0,0 +1,62 @@
+"k anal/meta/0x50=\0"
```

Comment:
@radare 

> * use “charset” rather than “encoding”
do you mean I put them into libr/util/d/charset?

> * make a simple C api to resolve that char instead of sdb
do you want that I make a new file radare2/libr/anal/charset.c

> * loading the byte=char instead of running an r2 script
do you expect I parse it with radare2/libr/anal/charset.c ?

> * what about multibyte charsets?
no idea what I can do

Reply:
dont put those files in libr/enco, this module shouldne exist at all. imho charset encodings should be in libr/util/d

charsets are part of text processing stuff, not analysis

Reply:
@radare  what do you mean by "make a simple C api to resolve that char instead of sdb" I can not write it in libr/enco/d/pokered it is a config file.

Reply:
Do you have an example of line that should be in pokered please?

Reply:
i think that we dont need to use sdb to store this information we can just expose a function in r_util to encode and decode passing a transformation function which represents the encoding. this function will take every char and convert them to ascii (or viceversa), we can hardcode those gameboy encodings and maybe other mame ones for now. and then add an api to dynamically load others via plugins.